File: Projects/React-Project/ems-app - Copy/src/model/Employee.js
export default class Empolyee{
    empId;
    empName;
    empSalary;

    constructor(){
        this.empId=0
        this.empName="";
        this.empSalary=0
    }
    setId(id){
        this.empId=id
    }
    setName(name){
        this.empName=name
    }
    setSalary(salary){
        this.empSalary=salary
    }
}

File: SpringBoot1/SB-EMS-Map/src/main/java/com/ems/model/Employee.java
package com.ems.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "employee")
public class Employee {

	@Id
	@Column(name = "empid")
	private int empId;
	
	@Column(name = "empname", length = 20)
	private String empName;
	
	@Column(name = "empsalary")
	private int empSalary;

	@Override
	public String toString() {
		return "Employee [empId=" + empId + ", empName=" + empName + ", empSalary=" + empSalary + "]";
	}

	public int getEmpId() {
		return empId;
	}

	public void setEmpId(int id) {
		this.empId = id;
	}

	public String getEmpName() {
		return empName;
	}

	public void setEmpName(String empName) {
		this.empName = empName;
	}

	public int getEmpSalary() {
		return empSalary;
	}

	public void setEmpSalary(int empSalary) {
		this.empSalary = empSalary;
	}
}
